mtgjson5/providers/tcgplayer.py

```diff
@@ -478,6 +479,16 @@ def generate_mtgjson_sealed_product_objects(
 
         sealed_data = get_tcgplayer_sealed_data(group_id)
 
+        # adjust for worlds decks
```

Comment:
id expand the comment "by looking at the last two digits being present in the product name"